src/Context/Auth/index.jsx

```diff
@@ -10,12 +11,19 @@ function AuthProvider({ children }){
   const [user, setUser] = useState({});
   const [error, setError] = useState(null);
 
+  React.useEffect(() => {
+    const cookieToken = cookie.load('auth');
+    _validateToken(cookieToken);
+  }, []);
```

Comment:
When you log out, make sure you remove the cookie. Right now if you log out then refresh the page, it will automatically log you back in.